projects/cps-ui-kit/src/lib/components/cps-file-upload/cps-file-upload.component.html

```diff
@@ -28,6 +29,9 @@
         extensionsStringAsterisks ? '(' + extensionsStringAsterisks + ')' : ''
       }}</span
     >
+    <div class="cps-file-upload-dropzone-content">
```

Comment:
I don't like the idea of providing the information about files using `ng-content`.
It is weird and unclear why it is provided differently from the other contents of the rectangle.

I would suggest solving this by using a prop `@Input() fileInfo = "";`
 
```html
@if (fileInfo) {
    <cps-icon color="calm" icon="info-circle" size="xsmall"></cps-icon>
    <span>{{fileInfo}}</span>
}
``` 
or alternatively if html format matters (to preserve breaks, etc.):
```html
<span [innerHTML]="fileInfo"></span>
```